foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/model/OpenAiChatCompletionStream.java

```diff
@@ -0,0 +1,28 @@
+package com.sap.ai.sdk.foundationmodels.openai.model;
+
+import java.util.concurrent.Future;
+import java.util.stream.Stream;
+import javax.annotation.Nonnull;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Getter
+@RequiredArgsConstructor(access = AccessLevel.NONE)
+@NoArgsConstructor
+@Setter
+@Accessors(chain = true)
+public class OpenAiChatCompletionStream<T> implements AutoCloseable {
```

Comment:
(Comment/Major/Discussion)

I think this class is an unnecessary wrapper. If we just exposed `Stream` we wouldn't need to care about `AutoCloseable` ourselves.